Assets/Source/Entities/Behaviors/Harvester.cs

```diff
@@ -0,0 +1,76 @@
+﻿using UnityEngine;
+using System.Collections;
+
+public class Harvester : MonoBehaviour
+{
+	// Stats
+	public float HarvestingRate;
+	public float HarvestRange;
+	public float DepositRange;
+	public float DeposityRetryRate;
```

Comment:
Hmm... DeposityRetryRate?

Did you mean DepositRetryRate? Or is this a feature?